src/resources/common.yml

```diff
@@ -188,8 +179,16 @@ common:
     en: Title
     de: Titel
   slug:
-    en: URL path
-    de: URL-Pfad
+    en: Slug
+    de: Slug
+  slugInfo:
+    en: |
+      The slug is added to the end of the URL and can contain one or more parts separated by slashed. Each part and can only contain lower case letters (a-z), digits and hyphens.
+    de: |
+      Der Slug wird am ende des URLs hinzugefügt und darf einen order mehrere durch Schrägstriche getrennte Teile enthalten. Jeder Teil darf nur Kleinbuchstaben (a-z), Ziffern und Bindestriche enthalten.
```

Comment:
Some typos: "**E**nde" / "o**~~r~~**der"

Reply:
Amazed those were the only problems! Corrected, thanks 😊